src/khoj/database/adapters/__init__.py

```diff
@@ -185,11 +188,29 @@ async def aget_or_create_user_by_email(email: str) -> tuple[KhojUser, bool]:
 
     user_subscription = await Subscription.objects.filter(user=user).afirst()
     if not user_subscription:
-        await Subscription.objects.acreate(user=user, type="trial")
+        await Subscription.objects.acreate(user=user, type="standard")
```

Comment:
The subscription for users created via Google OAuth in the `create_user_by_google_token` function should also be updated to `type="standard"`. It's currently set to type="trial"

Reply:
Good catch!